packages/boot/.travis.yml

```diff
@@ -0,0 +1,5 @@
+sudo: false
```

Comment:
Ditto, can we use top-level `.travis.yml` from loopback-next monorepo and remove this file?